docs/README.rst

```diff
@@ -102,69 +109,75 @@ These configuration parameters are global to the controller.
 |                    |         |          |             | handle OpenShift Route objects.         |                |
 |                    |         |          |             |                                         |                |
-|                    |         |          |             | Only applicable in the OpenShift        |                |
-|                    |         |          |             | environment.                            |                |
+|                    |         |          |             | **Only applicable in OpenShift.**       |                |
 +--------------------+---------+----------+-------------+-----------------------------------------+----------------+
 | route-vserver-addr | string  | Optional | n/a         | Bind address for virtual server for     |                |
 |                    |         |          |             | OpenShift Route objects.                |                |
 |                    |         |          |             |                                         |                |
-|                    |         |          |             | Only applicable in the OpenShift        |                |
-|                    |         |          |             | environment.                            |                |
+|                    |         |          |             | **Only applicable in OpenShift.**       |                |
 +--------------------+---------+----------+-------------+-----------------------------------------+----------------+
 | route-label        | string  | Optional | n/a         | Tells the ``k8s-bigip-ctlr`` to only    |                |
 |                    |         |          |             | watch for OpenShift Route objects with  |                |
-|                    |         |          |             | a label named 'f5type' set to the       |                |
-|                    |         |          |             | specified value.                        |                |
+|                    |         |          |             | the ``f5type`` label set to this value. |                |
 |                    |         |          |             |                                         |                |
-|                    |         |          |             | Only applicable in the OpenShift        |                |
-|                    |         |          |             | environment.                            |                |
+|                    |         |          |             | **Only applicable in OpenShift.**       |                |
 +--------------------+---------+----------+-------------+-----------------------------------------+----------------+
 
 
-VirtualServer ConfigMap Properties
-----------------------------------
-The |kctlr-long| supports VirtualServer ConfigMap objects.
+.. _f5 resource configmap properties:
```

Comment:
Is this a ref? I can't find where its being used.

Reply:
It might not be in use at present. It doesn't hurt anything by being there, though (IMO).